src/locale/translations/en.json

```diff
@@ -546,28 +547,28 @@
     },
     "ScanAPlace": {
       "Title": "Add a visit: Mooseville only",
-      "Body": "Get notified if someone from today tests positive later.\n\nThe app saves your visits on your phone, including the dates and times you were there.\n\n**Your visits are not shared** with anyone.",
-      "Banner": "Right now, posters with QR codes are only available in Mooseville as part of a small-scale pilot project.",
+      "Body": "Get notified if public health finds an exposure risk from your visit.\n\nThe app saves your visits on your phone, including the dates and times you were there.\n\n**Your visits are not shared** with anyone.",
+      "Banner": "Right now, posters with QR codes are **only available in Mooseville** as part of a small-scale pilot project.",
       "CTA": "Next",
       "CTA2": "About adding a visit"
     },
     "ProximityExposure": "Get tested",
     "OutbreakExposed": {
-      "Title": "Possible exposure at a visit",
-      "Body": "Someone tested positive from one of your visits.",
-      "DateDesc": "Date of exposure: ",
+      "Title": "You’ve been exposed",
+      "Body": "You visited a place at the same time as at least one person who tested positive for COVID-19.\n\nPublic health identified this exposure.",
+      "DateDesc": "Notification received: ",
```

Comment:
Let's hold off on this change until we have updated the logic - currently the date and all the other logic use the exposure date.